ts/src/pro/ascendex.ts

```diff
@@ -912,22 +923,30 @@ export default class ascendex extends ascendexRest {
         //     { m: 'sub', id: '1647515701', ch: 'depth:BTC/USDT', code: 0 }
         //
         const channel = this.safeString (message, 'ch', '');
-        if (channel.indexOf ('depth-realtime') > -1) {
-            this.handleOrderBookSubscription (client, message);
+        const id = this.safeString (message, 'id');
```

Comment:
changed this method (`handleSubscriptionStatus`) body to carry similar approach as we do on other exchanges.